src/frontend-scripts/components/section-main/replay/ReplayControls.jsx

```diff
@@ -135,16 +135,18 @@ const Description = ({ description }) => {
 const Playback = ({ hasNext, hasPrev, next, prev, forward, backward, beginning, end }) => {
 	const onKeyDown = event => {
 		// ignore typing in textboxes
+		const leftKeyCode = 37;
+		const rightKeyCode = 39;
 		if (['INPUT', 'TEXTAREA'].includes(document.activeElement.tagName)) return;
 		const char = String.fromCharCode(event.keyCode);
-		if (char === 'H' && hasPrev) {
-			backward();
-		} else if (char === 'J' && hasPrev) {
-			prev();
-		} else if (char === 'K' && hasNext) {
-			next();
-		} else if (char === 'L' && hasNext) {
-			forward();
+		if ((char === 'H' && hasPrev) || (event.shiftKey && event.keyCode == leftKeyCode)) {
```

Comment:
This will need the hasPrev and hasNext checks to be overarching for either of the keybinds - something like `(='H' || (shift && ='left')) && hasPrev)`

Reply:
thanks for catching this